src/main/java/creativei/entity/InquiryEducation.java

```diff
@@ -27,16 +27,17 @@
     @Column(name="mark_scheme")
     private MarkScheme markScheme;
     private String type;
+    public InquiryEducation(){}
 
     public InquiryEducation(EducationVo educationVo) {
         this.educationQualification=EducationQualification.stringToEnum(educationVo.getEducationQualification());
-        this.instituteName=educationVo.getInstituteName()==null?null:educationVo.getInstituteName();
+        this.instituteName=educationVo.getInstituteName()==null||educationVo.getInstituteName().isEmpty()?null:educationVo.getInstituteName();
         this.stream=Stream.stringToEnum(educationVo.getStream());
-        this.status=educationVo.getStatus()==null?null:Status.stringToEnum(educationVo.getStatus());
+        this.status=Status.stringToEnum(educationVo.getStatus());
         this.year=educationVo.getYear()==null?null:educationVo.getYear();
         this.aggregateMarks=educationVo.getAggregateMarks()==null?null:educationVo.getAggregateMarks();
-        this.markScheme=educationVo.getMarkScheme()==null?null:MarkScheme.stringToEnum(educationVo.getMarkScheme());
```

Comment:
tertiary not required here

Reply:
this.status=Status.stringToEnum(educationVo.getStatus()); 
this is the statement i have used it checks both for the null and empty string

Reply:
ok